cinemy/cinemy/settings.py

```diff
@@ -42,6 +42,7 @@
     "core",
     "contact",
     "login",
+    "core.templatetags.filter",
```

Comment:
why did you add this here?

Reply:
I have created a custom filter which is transforming the time from integer value in minutes to H:m format.

Reply:
Removed it as it's not needed.